layers/meta-resin-artik/recipes-support/resin-init/resin-init-board.bbappend

```diff
@@ -1 +1,10 @@
 FILESEXTRAPATHS_append_artik10 := "${THISDIR}/${PN}"
+FILESEXTRAPATHS_append_artik53x := ":${THISDIR}/${PN}"
+
+SRC_URI = " \
```

Comment:
You should also have it here with: `SRC_URI_artik530s`

Reply:
done